code/modules/mob/living/carbon/human/human_damage.dm

```diff
@@ -212,9 +212,13 @@
 ////////////////////////////////////////////
 
 //Returns a list of damaged organs
-/mob/living/carbon/human/proc/get_damaged_organs(var/brute, var/burn)
+/mob/living/carbon/human/proc/get_damaged_organs(var/brute, var/burn, var/flag = "BOTH") // flag takes ORG, SYNTH, OR BOTH for consistancy

```

Comment:
Alright, I've reset the branch so I can do this cleanly, I'm going to add the organ flags to the flag defines file, then I'm going to make the default of the function's flag variable 3 so both flags are active if undefined. Sound good?